src/AzureExtension/Strings/en-US/Resources.resw

```diff
@@ -554,9 +554,9 @@
     <value>No default user logged in. Please login from the Settings page.</value>
     <comment>Error shown when no user name is sent to the extension.</comment>
   </data>
-  <data name="DevBox_NotRunningFailedKey" xml:space="preserve">
-    <value>The Dev Box isn't running. Please start the Dev Box and try again.</value>
-    <comment>Error shown when the operation fails due to the Dev Box not currently running.</comment>
+  <data name="DevBox_ErrorStateKey" xml:space="preserve">
+    <value>The Dev Box isn't in a valid state</value>

```

Comment:
Maybe we should say something like "{0} couldn't be started. Please try starting the Dev Box manually and try again."

where {0} is the name of the Dev Box